Kitodo/src/main/java/org/kitodo/production/services/file/FileService.java

```diff
@@ -1295,7 +1295,7 @@ private void repaginatePhysicalDivisions(Workpiece workpiece) {
         }
         for (PhysicalDivision physicalDivision : physicalDivisions) {
             if (Objects.isNull(physicalDivision.getOrderlabel())) {
-                physicalDivision.setOrderlabel(" - ");
+                physicalDivision.setOrderlabel("-");
```

Comment:
Should be:

    physicalDivision.setOrderlabel("uncounted");

See also: `org.kitodo.production.helper.metadata.pagination.PaginatorType.UNCOUNTED`